SDCAlertView/Source/SDCAlertControllerDefaultVisualStyle.m

```diff
@@ -44,6 +44,14 @@ - (UIFont *)messageLabelFont {
 	return [UIFont systemFontOfSize:13];
 }
 
+-(UIColor *)titleLabelColor {
+	return [UIColor blackColor];
+}
+
+-(UIColor *)messageLabelColor {
+	return [UIColor blackColor];
+}
+
```

Comment:
Nitpick: can you add a space between the `-` and `(UIColor *)` so it's consistent with the other methods?